### ISSUE TITLE
Min-Vid seekbar and timestamp, are not updated when fast forwarding from keyboard if previously the video was paused

### ISSUE THREAD

**Issue by Softvision-VladBacia** [meandavejustice/min-vid#396]
**[Affected versions]:**
-Firefox 45.0 and up
-Min-Vid 0.3.0-dev

**[Affected Platforms]:**
-All Windows

**[Prerequisites]:**
-Latest Min-Vid add-on version (0.3.0-dev) was previously installed on a clean profile.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
2. Navigate to YouTube.
3. Send a video to the mini player and choose to play it.
4. Pause the video.
5. Tap the right arrow keyboard button for a 5 seconds fast forward the video.

**[Expected result]:**
-The seek bar and timestamp accurately display the progress of the video while fast forwarding.

**[Actual result]:**
-The seek bar and timestamp progress are not changed while the video is on pause.

**[Additional Notes]:**
-The issue is also reproducible when going fast backwards.
-[Here](https://drive.google.com/open?id=0B6QogV2AJFKiMHBTcFdUM0w2MGM) is a screen recording of the issue.

**Comment by vickychijwani** [comment]
Looks like a duplicate of #387.

**Comment by Softvision-VladBacia** [comment]
In issue #387 the seekbar and timestamp are updated on key release. In this issue (#396), the seekbar and timestamp are not updated on key release while the video is on pause, they are updated only after the video has been unpaused. Maybe the root cause is the same but they are different scenarios, so different bugs.

**Comment by meandavejustice** [comment]
Confirming this is still an issue as of today

**Comment by SoftVision-CiprianMuresan** [comment]
This issue is no longer reproducible because #757 disabled YouTube hotkeys.
